internal/util/duration_test.go

```diff
@@ -0,0 +1,135 @@
+package util
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExtendedParseDuration(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected time.Duration
+		wantErr  bool
+	}{
+		// Standard Go durations should work
+		{
+			name:     "standard seconds",
+			input:    "30s",
+			expected: 30 * time.Second,
+			wantErr:  false,
+		},
+		{
+			name:     "standard minutes",
+			input:    "5m",
+			expected: 5 * time.Minute,
+			wantErr:  false,
+		},
+		{
+			name:     "standard hours",
+			input:    "2h",
+			expected: 2 * time.Hour,
+			wantErr:  false,
+		},
+		{
+			name:     "complex standard duration",
+			input:    "1h30m45s",
```

Comment:
These complex durations are not allowed by the current Duration openAPI spec `^[1-9]\d*[smh]$`, and therefore they are not supported in the UI.

I think we should stick to the single unit rule that we have now.